src/Employer/Employer.Web/Controllers/VacancyPreviewController.cs

```diff
@@ -89,6 +89,14 @@ public async Task<IActionResult> Review(SubmitReviewModel m)
         [HttpPost("preview", Name = RouteNames.Preview_Submit_Post)]
         public async Task<IActionResult> Submit(SubmitEditModel m)
         {
+            if (!ModelState.IsValid)
+            {
+                var viewModel1 = await _orchestrator.GetVacancyPreviewViewModelAsync(m);
```

Comment:
I know its done below, but can we have a single variable declaration for this please

Reply:
Amended in latest.